apiServices/src/main/app.py

```diff
@@ -816,7 +828,35 @@ def update_conditions(_id):
     except Exception as e:
         # Handle unexpected exceptions and return a generic error message
         return jsonify({"status": 500, "code": "Internal Server Error", "result": [{"message": "An error occurred"}]})
-        
+    
+
+@app.route('/template/api/v1/survey/getSolutions', methods=['POST'])
+def fetchSurveySolutions():
+    survey = surveyCreate()
+    access_token = survey.generate_access_token()
+    csv_path=survey.fetch_solution_id(access_token)
+    # print(accesstoken)
+    if csv_path:
+        return jsonify({"status": 200, "code": "Success","csvPath":csv_path})
+    
+    else:
+        return jsonify({"status": 500, "code": "NOTOK","csvPath":"Could not get csv path"})
+
+
+
+@app.route('/template/api/v1/survey/create', methods=['POST'])
+def create():
+    req = request.get_json()
+    helperInstance = Helpers
+   
+    print("=-=-=-=-=-=-=-=> ",req['file'])
+    resourceFile=helperInstance.loadSurveyFile(req['file'])
+    print(resourceFile,"resourceFileresourceFile")
+    if resourceFile:
+        return jsonify({"status": 200, "code": "Success", "result": [{"solutionId":resourceFile[0],"successSheet":resourceFile[1],"downloadbleUrl":resourceFile[2]}]})
```

Comment:
Don't use list here resourceFile[1] ,  resourceFile[2] etc...
use a dictionary resourceFile['successSheet']. It improves the readability of the code.